src/Appwrite/Platform/Modules/Sites/Http/Deployments/Download/Get.php

```diff
@@ -31,7 +31,6 @@ public function __construct()
         $this
             ->setHttpMethod(Action::HTTP_REQUEST_METHOD_GET)
             ->setHttpPath('/v1/sites/:siteId/deployments/:deploymentId/download')
-            ->httpAlias('/v1/sites/:functionId/deployments/:deploymentId/build/download', [ 'type' => 'output' ])
```

Comment:
This was added yesterday since the default value was `source`. Why did we remove it?

Reply:
Ah, right